release_notes/41-release-notes.adoc

```diff
@@ -0,0 +1,438 @@
+=== Launching Vulnerability Management 2.0 (Technology Preview)
+
+In {product-title-short} 4.1, you can experience significant improvements in Vulnerability Management with the introduction of Workload CVEs (Technology Preview). This feature is the first step towards Vulnerability Management 2.0 and provides you with improved workflows that are purposeful and effortless to navigate.
+
+These improved workflows provide actionable data with just a few clicks to help you address vulnerabilities. It is anticipated that Vulnerability Management 1.0 functionality will gradually transition to Vulnerability Management 2.0 in upcoming releases.
+
+This release provides a new navigation menu item, Vulnerability Management 2.0, which currently provides a view of vulnerabilities in applications running on clusters in your system. It is anticipated that in upcoming releases, this menu item will also provide a view of vulnerabilities present in platform and nodes.
+
+The *Workload CVEs* page provides a view of vulnerabilities in images and deployments, as well as options to filter and sort the data displayed. You can select specific vulnerabilities to view additional details, such as the Dockerfile layer in which the vulnerability exists and whether or not it can be fixed.
+
+In this technology preview, you can explore the read-only *Workload CVEs* page. However, upcoming releases are expected to include additional functionality such as Reporting, Risk Acceptance, and Manage Watched Images functionality. For more information, see xref:../operating/manage-vulnerabilities/vulnerability-management.adoc#vulnerability-management-process_acs-operating-manage-vulnerabilities[Vulnerability Management process].
+
+:FeatureName: Vulnerability Management 2.0
+include::snippets/technology-preview.adoc[]
+
+
+[id="proxy-configuration-for-secured-cluster-services-on-rhacs-cloud-service_{context}"]
+=== Proxy configuration for secured cluster services on {product-title-managed-short}
+
+With this release, you can establish a reliable connection between the secured cluster and the designated proxy server, ensuring seamless data collection and transmission.
+This improves the overall security and performance of your {rh-rhacscs-first} environment and enables effective monitoring and protection of your cluster resources.
+
+With this feature, you can set the `HTTP_PROXY` and `HTTPS_PROXY` variables to define the proxy server addresses for `HTTP` and `HTTPS` connections, respectively.
+You can also use the `NO_PROXY` variable to specify hostnames or IP addresses that should not be accessed through the proxy server. For more information, see xref:../cloud_service/installing_cloud_ocp/configuring-the-proxy-for-secured-cluster-services-in-rhacs-cloud-service.adoc#configuring-the-proxy-for-secured-cluster-services-in-rhacs-cloud-service[Configuring the proxy for secured cluster services in {product-title-managed-short}].
```

Comment:
@rukletsov I think it is better we keep this section since it is related to https://issues.redhat.com/browse/ROX-14958. Please can you suggest a more appropriate which highlights the business impact/what issue is solved for the customer, followed by what this feature offers?, thx!

Reply:
But https://issues.redhat.com/browse/ROX-14958 only documented what we already had for I think 2 years. I think selling this as a new feature is confusing.

Reply:
@rukletsov ok, since we have documented a new section can we move this then to the "Documentation additions" section (I need to create this section). WDYT?

Reply:
We could. Note that it will require a substantial rewrite then. 

> I need to create this section
Could you please satisfy my curiosity and tell me why? We update documentation and fix bugs on a regular basis, so why this one requires being called out?

Reply:
@rukletsov Ah, I might have gotten confused here because of https://docs.openshift.com/acs/4.0/release_notes/40-release-notes.html#doc-additions-rn40
If you really feel that we don't need to call this out, let us remove it from the Release notes. Apologies for creating confusion.

Reply:
@kcarmichael08 Your thoughts, please!

Reply:
So my understanding is that how to configure proxies wasn't documented, but that info was requested by customers, so we added it to the docs. We will sometimes call out documentation updates/changes in the RNs if they were high visibility, requested by customers, etc. So I could make a case for putting in a short item - "updated documentation to include instructions for configuring....." **or** I could make the case that we're just adding something that was missing in the docs, and it doesn't need to be called out. Since we're running out of time and it was just one customer that requested it, I would say we could remove it. Anyone who needs to point the customer to it can point them directly to the docs.

Reply:
> @rukletsov Ah, I might have gotten confused here because of [docs.openshift.com/acs/4.0/release_notes/40-release-notes.html#doc-additions-rn40](https://docs.openshift.com/acs/4.0/release_notes/40-release-notes.html#doc-additions-rn40) If you really feel that we don't need to call this out, let us remove it from the Release notes. Apologies for creating confusion.

Yeah, we don't normally note updates to the docs unless we really want to make customers aware of them. Mostly things get quietly changed and updated in docs without a mention in the RNs. :)

Reply:
Thanks for sharing your thoughts, Kerry!
I am gonna remove this section from the RN.